cmd/plume/types.go

```diff
@@ -78,3 +78,21 @@ type channelSpec struct {
 	Azure        azureSpec
 	AWS          awsSpec
 }
+
+type fcosChannelSpec struct {
+	Bucket  string
+	Profile string
+	Region  string
+}
+
+type ReleaseMetadata struct {
+	Observation string          `json:"observation"` // used to note to users not to consume the release metadata index
```

Comment:
Feel free to bikeshed this name :)

Reply:
`info`?  `note`?  `caution`?

Reply:
I like `note` but I could be swayed to `caution` if we feel it better conveys that users should pay attention to it :)